.github/container/upstream-paxml-requirements.in

```diff
@@ -0,0 +1,6 @@
+-c jax-requirements.in
+/opt/paxml
+/opt/praxis
+# Re-introduce the dependency that was skipped by SKIP_HEAD_INSTALLS=true
+# Omitting jax since it is included by the jax-requirements.in constraint
+fiddle @ git+https://github.com/google/fiddle
```

Comment:
Is fiddle pinned?

Reply:
It is. The logic to pin it is here: https://github.com/NVIDIA/JAX-Toolbox/pull/271/files#diff-8386cfe5c29a3be212a911c9550a56c57420a17cae190b80f60e52c0ce265645R151-R175

fiddle will appear as `fiddle @ git+https://github.com/google/fiddle` in the `*requirements.txt` file after `pip-compile` is run; and after [this](https://github.com/NVIDIA/JAX-Toolbox/pull/271/files#diff-8386cfe5c29a3be212a911c9550a56c57420a17cae190b80f60e52c0ce265645R151-R175) is run, it will look like `fiddle @ git+https://github.com/google/fiddle@$COMMIT_SHA`